rule-engine/rule-engine-components/src/main/java/org/thingsboard/rule/engine/filter/TbCheckRelationNodeConfiguration.java

```diff
@@ -30,13 +30,15 @@ public class TbCheckRelationNodeConfiguration implements NodeConfiguration<TbChe
     private String entityType;
     private String relationType;
     private boolean checkForSingleEntity;
+    private boolean tbCheckRelationNodeV2;
 
     @Override
     public TbCheckRelationNodeConfiguration defaultConfiguration() {
         TbCheckRelationNodeConfiguration configuration = new TbCheckRelationNodeConfiguration();
         configuration.setDirection(EntitySearchDirection.FROM.name());
         configuration.setRelationType("Contains");
         configuration.setCheckForSingleEntity(true);
+        configuration.setTbCheckRelationNodeV2(true);
```

Comment:
What if the person will load the new version of the code with old UI? Are you sure that after save the parameter is always equal to v2? If UI will accidentally miss this parameter in the request?